vmngclient/api/dashboard_api.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from vmngclient.typed_list import DataSequence
+from vmngclient.utils.creation_tools import create_dataclass
+from vmngclient.utils.dashboard import CertificatesStatus, Count
+
+if TYPE_CHECKING:
```

Comment:
A special constant that is assumed to be True by 3rd party static type checkers. It is False at runtime